egs/aesrc2020/asr1/conf/tuning/train_pytorch_conformer_kernel15.yaml

```diff
@@ -0,0 +1,47 @@
+# network architecture
```

Comment:
I would suggest we keep the one best config or configs for each model architecture.

Reply:
I'm running some models. Will update README and confs after that finishes.

Reply:
Sounds very cool!